ipi_tests/examples/exampletools.py

```diff
@@ -84,15 +98,21 @@ def find_examples(parent, excluded_file="excluded_test.txt"):
         if os.path.isfile(Path(ff) / "input.xml"):
             if ff not in excluded:
                 examples.append(ff)
+                print(u"\u2713", " ", ff)
```

Comment:
Why is this necessary?

Reply:
I removed this, it wasn't necesssary, I just had it there for debugging checks.